lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
+ *
+ * @param {!Node} element
```

Comment:
Just re ran the build and looked more closely. The issue is that because this function calls itself to drill down and find nested tags, the nested nodes are not guaranteed to be elements. Initial call is always !Element but subsequent calls are not guaranteed.

```
/Users/sanbornh/Documents/shaka-player/lib/media/ttml_text_parser.js:223: ERROR - actual parameter 1 of shaka.media.TtmlTextParser.addNewLines_ does not match formal parameter
found   : Node
required: Element
      shaka.media.TtmlTextParser.addNewLines_(childNodes[i]);
```